TMGToolbox/src/input_output/import_binary_matrix.py

```diff
@@ -182,9 +204,17 @@ def __call__(self, xtmf_MatrixType, xtmf_MatrixNumber, ImportFile, xtmf_Scenario
     def _Execute(self):
         with _m.logbook_trace(name="%s v%s" %(self.__class__.__name__, self.version), \
                               attributes= self._GetAtts()):
-            matrix = _util.initializeMatrix(self.MatrixId)
-            if self.MatrixDescription:
-                matrix.description = self.MatrixDescription
+
+            if self.MatrixId == None:
+                new_matrix_id = "mf" + str(self.NewMatrixID)
+                new_matrix_name = self.NewMatrixName
+                new_matrix_desc = self.NewMatrixDescription
+                _CreateMatrixTool(matrix_id=new_matrix_id, matrix_name=new_matrix_name, matrix_description=new_matrix_desc)
+                matrix = _util.initializeMatrix(self.NewMatrixID)
```

Comment:
Either this, or the above 4 lines are redundant. `tmg.common.utilities.initializeMatrix` creates matrices and also accepts descriptions.

Reply:
Changed as suggested.